dali/python/nvidia/dali/pipeline.py

```diff
@@ -218,6 +236,33 @@ def __init__(self, batch_size = -1, num_threads = -1, device_id = -1, seed = -1,
         else:
             raise TypeError("Expected prefetch_queue_depth to be either int or Dict[int, int]")
 
+        # Assign and validate output_dtype
+        if type(output_dtype) is list:
+            for dtype in output_dtype:
+                if type(dtype) is not types.DALIDataType and dtype is not None:
+                    raise TypeError(
+                        f"`output_dtype` must be either: a type from nvidia.dali.types.DALIDataType, a list of these or None. Found type {type(dtype)} in the list.")
```

Comment:
Hmm, technically those are enum values :)
```suggestion
                        f"`output_dtype` must be either: None, a value of nvidia.dali.types.DALIDataType type or a list of these. Found type {type(dtype)} in the list.")
```
I know what you mean, but maybe something like this, or similar? (I also changed the order a bit, maybe it matches the doc a bit better).

Same below.

Reply:
Done